source/release-notes/1.6.txt

```diff
@@ -0,0 +1,31 @@
+.. _c2c-release-notes-1.6:
+
+===================================
+Release Notes for mongosync 1.6
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.6.0-c2c-release-notes:
+
+1.6.0 Release
+-------------
+
+**Upcoming**
+
+7.0 Support
+~~~~~~~~~~~
+
+Starting in 1.6.0, ``mongosync`` supports sync with MongoDB 7.0.
```

Comment:
I seem to recall that there was some discussion that we want to use "synchronization" instead of the shortened "sync". I could be misremembering, though.

Reply:
There was, but they came out in favor of sync.

Reply:
I wasn't a part of the earlier discussion but I'd prefer"sync" since we use that over "synchronization" in our public documentation (ex: in this [page](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/mongosync/))